BCP00301Test.py

```diff
@@ -42,8 +42,8 @@ def perform_test_ssl(self, test, args=None):
             ret = subprocess.run(["testssl/testssl.sh", "--jsonfile", TMPFILE, "--warnings", "off"] + args +
                                  ["{}:{}".format(self.apis[BCP_API_KEY]["hostname"], self.apis[BCP_API_KEY]["port"])])
         except Exception as e:
-            raise NMOSTestException(test.FAIL("Unable to execute testssl.sh. Please see the README for installation "
-                                              "instructions: {}".format(e)))
+            raise NMOSTestException(test.DISABLED("Unable to execute testssl.sh. Please see the README for installation"
+                                                  " instructions: {}".format(e)))

```

Comment:
LOL, we need to agree a coding style for spaces of strings like this, because I've been changing ones that I touch to trailing on the previous line, not leading on the subsequent line! :-)

Reply:
Ah, I hadn't been aiming for quite that level of consistency!